cabby/geo/walk.py

```diff
@@ -541,8 +581,10 @@ def generate_and_save_rvs_routes(path: Text, map: map_structure.Map, n_samples:
     columns=['osmid', 'geometry', 'main_tag'])
 
   counter = 0
+  if n_samples > 1:
+    sample_rand = True
```

Comment:
This doesn't seem like a clear contract with callers of this function (e.g. I might want a single sample, but to have it picked randomly). I would add an argument that allows them to  specify that they want a deterministic output.

Reply:
Changed